.ci-operator.yaml

```diff
@@ -1,4 +1,4 @@
 build_root_image:
   name: release
   namespace: openshift
-  tag: rhel-8-release-golang-1.17-openshift-4.10
+  tag: golang-1.19
```

Comment:
Can you change this to `rhel-9-golang-1.19-openshift-4.13` ?

Reply:
`rhel-9-golang-1.19-openshift-4.13` tag isn't available yet, which is the same reason why I used just `golang-1.19`.

Reply:
JFYR:

```
$ skopeo list-tags docker://registry.ci.openshift.org/openshift/release
{
    "Repository": "registry.ci.openshift.org/openshift/release",
    "Tags": [
        "tectonic-console-builder-v24",
        "golang-1.18",
        "rhel-7-release-openshift-4.11",
        "tectonic-console-builder-v21",
        "rhel-8-release-golang-1.19-openshift-4.13",
        "jdk-8",
        "rhel-7-release-openshift-4.6",
        "rhel-7-release-openshift-4.7",
        "rhel-8-release-golang-1.15-openshift-4.7",
        "rhel-8-release-golang-1.18-openshift-4.12",
        "tectonic-console-builder-v23",
        "golang-1.19",
        "golang-1.16-vault-typescript",
        "golang-1.17",
        "rhel-golang-1.12",
        "tectonic-console-builder-v20",
        "golang-1.16",
        "rhel-8-release-golang-1.19-openshift-4.12",
        "rhel-7-release-openshift-4.6.art",
        "nodejs-8-browser-tests",
        "rhel-7-release-openshift-4.10",
        "rhel-8-release-golang-1.16-openshift-4.10",
        "tectonic-console-builder-v17",
        "golang-1.11",
        "rhel-8-release-golang-1.16-openshift-4.8",
        "rhel-8-release-openshift-4.6",
        "tectonic-console-builder-v22",
        "rhel-7-release-openshift-4.9",
        "rhel-7-golang-1.11",
        "golang-1.14",
        "golang-1.16-vault",
        "rhel-8-release-golang-1.14-openshift-4.7",
        "rhel-8-release-golang-1.15-openshift-4.8",
        "tectonic-console-builder-v18",
        "tectonic-console-builder-v19",
        "golang-1.15",
        "rhel-8-release-golang-1.18-openshift-4.11",
        "ruby-25",
        "rhel-8-release-golang-1.17-openshift-4.10",
        "rhel-7-release-openshift-4.8.art",
        "golang-1.9",
        "golang-1.8",
        "rhel-8-release-golang-1.14-openshift-4.8",
        "rhel-8-release-openshift-4.3",
        "golang-1.10",
        "nodejs-8",
        "rhel-7-golang-1.12",
        "rhel-7-release-openshift-4.7.art",
        "rhel-8-release-golang-1.14-openshift-4.6",
        "rhel-8-release-golang-1.17-openshift-4.11",
        "rhel-golang-1.11",
        "golang-1.12",
        "rhel-7-release-openshift-4.8",
        "rhel-8-release-golang-1.15-openshift-4.6",
        "rhel-8-release-golang-1.16-openshift-4.9",
        "golang-1.13"
    ]
}
```

Reply:
alternative (apart from this) is to use: `registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.19-openshift-4.13` similar to what is in: [o/oc-mirror:.ci-operator.yaml](https://github.com/openshift/oc-mirror/blob/main/.ci-operator.yaml).

Reply:
```
podman pull registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.19-openshift-4.13
Trying to pull registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.19-openshift-4.13...
Getting image source signatures
Copying blob fda3ab9c84ea [=============>------------------------] 54.9MiB / 150.1MiB
Copying blob 793a8691df4a [========================>-------------] 49.4MiB / 75.2MiB
Copying blob 70e0ef827acf [==>-----------------------------------] 46.8MiB / 574.5MiB
```
Seems to be available. 

Also update corresponding pull specs in release.yaml,
- https://github.com/openshift/release/blob/7b17bfb10eb58bf18cef1ccd877f4bc39d8f6ee0/ci-operator/config/openshift/cert-manager-operator/openshift-cert-manager-operator-master.yaml#L18
- https://github.com/openshift/release/blob/7b17bfb10eb58bf18cef1ccd877f4bc39d8f6ee0/ci-operator/config/openshift/cert-manager-operator/openshift-cert-manager-operator-master.yaml#L29

Reply:
yep, so changes from https://github.com/openshift/release/pull/35770 might have got reverted as our repo's master branch still got old tags.

As per discussion, changing the operator's .ci-operator.yaml, ci Dockerfile in separate PR: #100 which should be preceded for sync by https://github.com/openshift/release/pull/35870.